LocationList.py

```diff
@@ -239,10 +396,26 @@ def shop_address(shop_id, shelf_id):
     ("DMT GS Bean Patch",                                            ("GS Token",     0x0F,  0x02, None,                        'Gold Skulltula Token',                  ("Death Mountain Trail", "Skulltulas",))),
     ("DMT GS Above Dodongos Cavern",                                 ("GS Token",     0x0F,  0x08, None,                        'Gold Skulltula Token',                  ("Death Mountain Trail", "Skulltulas",))),
     ("DMT GS Falling Rocks Path",                                    ("GS Token",     0x0F,  0x10, None,                        'Gold Skulltula Token',                  ("Death Mountain Trail", "Skulltulas",))),
+    # Death Mountain Trail Freestanding
+    ("DMT Cow Grotto Green Rupee 1",                   ("Collectable",0x3E,  0x40, ([0x026D2098], None),                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
+    ("DMT Cow Grotto Green Rupee 2",                   ("Collectable",0x3E,  0x41, None,                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
+    ("DMT Cow Grotto Green Rupee 3",                   ("Collectable",0x3E,  0x42, None,                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
+    ("DMT Cow Grotto Green Rupee 4",                   ("Collectable",0x3E,  0x43, None,                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
+    ("DMT Cow Grotto Green Rupee 5",                   ("Collectable",0x3E,  0x44, None,                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
+    ("DMT Cow Grotto Green Rupee 6",                   ("Collectable",0x3E,  0x45, None,                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
+    ("DMT Cow Grotto Red Rupee",                       ("Collectable",0x3E,  0x46, None,                        'Rupees (20)',                          ("Death Mountain Trail", "Death Mountain", "Grottos", "RupeeTower"))),
```

Comment:
Minor tabbing issue

Reply:
I reformatted the entirety of LocationList.py so this should be good now.